### ISSUE TITLE
Windows mauve lowestOneBit unexpected result

### ISSUE THREAD

**Issue by pshipton** [eclipse/openj9#7435]
https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_special.system_x86-64_windows_Nightly/426
MauveMultiThreadLoadTest_special_26
variation: Mode612-OSRG
JVM_OPTIONS: -Xcompressedrefs -Xcompressedrefs -Xgcpolicy:gencon -Xjit:enableOSR,enableOSROnGuardFailure,count=1,disableAsyncCompilation 
```
LT  06:08:57.954 - Test failed
LT    Failure num.  = 1
LT    Test number   = 1846
LT    Test details  = 'Mauve[gnu.testlet.java.lang.Long.Tests15]'
LT    Suite number  = 0
LT    Thread number = 3
LT  >>> Captured test output >>>
LT  PASS: gnu.testlet.java.lang.Long.Tests15 (number 0)
LT  PASS: gnu.testlet.java.lang.Long.Tests15 (number 1)
LT  PASS: gnu.testlet.java.lang.Long.Tests15: bitCount (number 0)
LT  PASS: gnu.testlet.java.lang.Long.Tests15: bitCount (number 1)
LT  PASS: gnu.testlet.java.lang.Long.Tests15: bitCount (number 2)
LT  PASS: gnu.testlet.java.lang.Long.Tests15: bitCount (number 3)
LT  PASS: gnu.testlet.java.lang.Long.Tests15: bitCount (number 4)
LT  PASS: gnu.testlet.java.lang.Long.Tests15: rotateLeft (number 0)
LT  PASS: gnu.testlet.java.lang.Long.Tests15: rotateLeft (number 1)
LT  PASS: gnu.testlet.java.lang.Long.Tests15: rotateRight (number 0)
LT  PASS: gnu.testlet.java.lang.Long.Tests15: rotateRight (number 1)
LT  PASS: gnu.testlet.java.lang.Long.Tests15: highestOneBit (number 0)
LT  PASS: gnu.testlet.java.lang.Long.Tests15: highestOneBit (number 1)
LT  PASS: gnu.testlet.java.lang.Long.Tests15: highestOneBit (number 2)
LT  PASS: gnu.testlet.java.lang.Long.Tests15: numberOfLeadingZeros (number 0)
LT  PASS: gnu.testlet.java.lang.Long.Tests15: numberOfLeadingZeros (number 1)
LT  PASS: gnu.testlet.java.lang.Long.Tests15: numberOfLeadingZeros (number 2)
LT  PASS: gnu.testlet.java.lang.Long.Tests15: numberOfLeadingZeros (number 3)
LT  FAIL: gnu.testlet.java.lang.Long.Tests15: lowestOneBit (number 0)
LT  got 36028797018963968 but expected 1
LT  FAIL: gnu.testlet.java.lang.Long.Tests15: lowestOneBit (number 1)
LT  got -9223372036854775808 but expected 1
LT  FAIL: gnu.testlet.java.lang.Long.Tests15: lowestOneBit (number 2)
LT  got -9223372036854775808 but expected 256
```
https://ci.eclipse.org/openj9//job/Grinder/parambuild/?JDK_VERSION=8&JDK_IMPL=openj9&BUILD_LIST=system/mauveLoadTest&JenkinsFile=openjdk_x86-64_windows&TARGET=MauveMultiThreadLoadTest_special_26&SDK_RESOURCE=upstream&CUSTOMIZED_SDK_URL=https://140-211-168-230-openstack.osuosl.org/artifactory/ci-eclipse-openj9/Build_JDK8_x86-64_windows_Nightly/172/OpenJ9-JDK8-x86-64_windows-20191011-022235.tar.gz%20https://140-211-168-230-openstack.osuosl.org/artifactory/ci-eclipse-openj9/Build_JDK8_x86-64_windows_Nightly/172/native-test-libs.tar.gz&CUSTOMIZED_SDK_URL_CREDENTIAL_ID=ab89294b-5ba1-48e9-8c85-107daca5a2e9

**Comment by pshipton** [comment]
20x grinder https://ci.eclipse.org/openj9/job/Grinder/535/

**Comment by pshipton** [comment]
Closed as a dup of #7088
